lib/fastlane/plugin/wpmreleasetoolkit/models/build_number.rb

```diff
@@ -0,0 +1,38 @@
+require 'xcodeproj'
+
+module Fastlane
+  module Model
+    class BuildNumber
```

Comment:
This class doesn't model a build number so much as it does a file _containing_ a build number among other things (and seems very iOS-specific).

If you're trying to model a build number (and the operations that can be performed on it), you'll probably never want your model to be able to persist itself. That's generally true for anything that isn't modelling a file – if your class is called `ConfigurationFile`, it might make sense to have a `save` or `write` method, but for something that's consolidating computation like this it's probably not quite what you're after.